_test_common/lib/assets.dart

```diff
@@ -13,10 +13,14 @@ AssetNode makeAssetNode([
   Digest? lastKnownDigest,
 ]) {
   var id = makeAssetId(assetIdString);
-  var node = AssetNode.source(id, lastKnownDigest: lastKnownDigest);
-  if (outputs != null) {
-    node.outputs.addAll(outputs);
-    node.primaryOutputs.addAll(outputs);
+  if (outputs == null) {
```

Comment:
Isn't this if unnecessary?
You return the same thing and the constructor takes null for the parameters you pass `outputs` in to below.

Reply:
Thanks, done.